cmd/registry/tests/task-validation_test.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/apigee/registry/cmd/registry/cmd"
+	"github.com/apigee/registry/cmd/registry/controller"
+	"strings"
+	"testing"
+)
+
+// Test that GetCommand() is returning the correct instance for each command string
+func TestValidateCommands(t *testing.T) {
```

Comment:
Nit: Conventionally, this should be `TestGetCommand` if it's going to test GetCommand directly.

Reply:
Done